OpenRA.Mods.RA/Traits/Infiltration/InfiltrateForActor.cs

```diff
@@ -13,18 +13,18 @@
 
 namespace OpenRA.Mods.RA.Traits
 {
-	class InfiltrateForSupportPowerInfo : ITraitInfo
+	class InfiltrateForActorInfo : ITraitInfo
```

Comment:
Two things:
1) Can this wait until I get feedback in the PR discussion on how to change the rest of the code? This trait will be renamed to InfiltrateForProxyActor (as @pchote preferred).

2) Running "OpenRA.Utility ra --upgrade-mod" gives
Received args: --upgrade-mod
System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at OpenRA.Mods.Common.UtilityCommands.UpgradeModCommand.Run(ModData modData, String[] args)
   at OpenRA.Utility.Program.Main(String[] args) in d:\Documents\Visual Studio 2013\Projects\OpenRA\OpenRA.Utility\Program.cs:line 54